configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>What about condition type? (FAILURE / INSIGHT)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Haven't dived into it here. don't you think it's a bit overkill?</p>